react/features/lobby/components/AbstractLobbyScreen.js

```diff
@@ -397,6 +404,9 @@ export function _mapStateToProps(state: Object): $Shape<Props> {
         _participantName: localParticipant?.name,
         _passwordJoinFailed: passwordJoinFailed,
         _renderPassword: !iAmSipGateway,
+        _toolbarButtons: isVpaaS
+            ? [ 'microphone', 'camera', 'select-background', 'settings' ]
```

Comment:
Extract these to constants to avoid re-renders.

Reply:
created selector for buttons/